Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -26,6 +28,7 @@ public class TextDocumentLoader : ITextDocumentLoader {
     // 256MB
     private const int MaxStackSize = 0x10000000;
```

Comment:
Why is this duplicated from above? (I know it's not from this patch.)

Reply:
Specifically the Dafny resolver and translator need larger stacks, but this is caused by independent problems that could be resolved independently, so I've also kept the large stack workarounds independent. The code duplication is tiny.